libs/sdk-ui-dashboard/src/model/state/dashboardStore.ts

```diff
@@ -191,6 +192,7 @@ export function createDashboardStore(config: DashboardStoreConfig): ReduxedDashb
         dateFilterConfig: dateFilterConfigSliceReducer,
         insights: insightsSliceReducer,
         alerts: alertsSliceReducer,
+        availableDrillTarget: availableDrillTargetReducer,
```

Comment:
i propose renaming.. to something like `widgetDrillTargets` or just `drillTargets` (prular is nicer here.. notice the rest is insights, alerts etc etc)

Reply:
I agree, the availability of the targets is implied by that they are there at all. I would go for `drillTargets` as I don't think things other than widgets make sense for drilling.

Reply:
All related stuff renamed thx for note.